src/app/app-modules/set-security-questions/set-security-questions.component.html

```
@@ -1,259 +1,244 @@
-              <mat-option
-                *ngFor="let question of questions"
-                [value]="question.QuestionID">
-                {{ question.Question }}
-              </mat-option>
-            </mat-select>
-          </mat-form-field>
-        </div>
-        <div class="col-xs-12 col-sm-12">
-          <mat-form-field style="width: 95%">
-            <mat-label>Answer 1</mat-label>
+<div class="support-wrapper">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)